lib/capybara/node/actions.rb

```diff
@@ -258,13 +260,16 @@ def unselect(value = nil, from: nil, **options)
       #   @param [String] locator                  Which field to attach the file to
       #   @param [String, Array<String>] paths     The path(s) of the file(s) that will be attached
       #
-      #   @option options [Symbol] match (Capybara.match)     The matching strategy to use (:one, :first, :prefer_exact, :smart).
-      #   @option options [Boolean] exact (Capybara.exact)    Match the exact label name/contents or accept a partial match.
+      #   @option options [Symbol] match
+      #     The matching strategy to use. Defaults to {Capybara.configure match}.
+      #   @option options [Boolean] exact
+      #     Match the exact label name/contents or accept a partial match. Defaults to {Capybara.configure exact}.
```

Comment:
[note] `(Capybara.match)` or `(Capybara.exact)` is generated as plain text, not link: ⬇️ 

![image](https://user-images.githubusercontent.com/473530/58217976-285be200-7d40-11e9-908d-ea565c9bb2bd.png)

Also, because one-line descriptions can break the vertical alignment easily and make less readable, I insert a newline for descriptions. But I welcome to any feedbacks for this. 😃

Reply:
Reality is most people actually reading the code aren't reading the comments used to generate the docs so one-line descriptions or not is not really an issue (comments are intentionally excluded from rubocop line length checks for this reason) - Overall though  don't mind either way.